shared-svcs-stamp.json

```
@@ -518,6 +533,32 @@
                 "enabledState": "Enabled",
                 "friendlyName": "[variables('frontDoorName')]"
             }
+        },
+        {
+            "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+            "apiVersion": "2022-01-31-preview",
+            "name": "ghActionFederatedIdentity",
+            "location": "[parameters('location')]",
+            "metadata": {
+                "description": "Create the Azure Credentials for the GitHub CD workflow"
+            },
+            "resources": [
+                {
+                    "type": "Microsoft.ManagedIdentity/userAssignedIdentities/federatedIdentityCredentials",
+                    "apiVersion": "2022-01-31-preview",
+                    "name": "[format('{0}/{1}', 'ghActionFederatedIdentity', 'ghActionFederatedIdentity')]",
```

The purpose of moving to a child resource is to avoid the complexity of naming them, in addition to showing clear intent/relationship.

```suggestion
                    "type": "federatedIdentityCredentials",
                    "apiVersion": "2022-01-31-preview",
                    "name": "ghActionFederatedIdentity",
```

---

https://github.com/mspnp/aks-baseline-multi-region/pull/72/commits/b0519e5207a51fa055658eff50be0ecb93c1d985